apptest/tester/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM gcr.io/cloud-marketplace-tools/testrunner:0.1.4
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+    curl wget \
+    && rm -rf /var/lib/apt/lists/*
+
+RUN mkdir -p /opt/kubectl/1.14 \
```

Comment:
this is pretty old Kubernetes, are there no later versions available? (I think its currently at 1.22 or 1.23 or so)